src/server/handlers/github.js

```diff
@@ -138,31 +144,19 @@ export const listRepositories = (req, res) => {
         });
       }
 
-      return Promise.all(response.body.map((repo) => {
```

Comment:
Removed the `snap_info` feature from `listRepositories`. The snap name is not needed in the "select repositories" list, and stripping this out should improve performance.